operator/pkg/gateway-api/grpcroute_reconcile.go

```diff
@@ -139,7 +139,7 @@ func (r *grpcRouteReconciler) updateStatus(ctx context.Context, original *gatewa
 
 func (r *grpcRouteReconciler) handleReconcileErrorWithStatus(ctx context.Context, reconcileErr error, original *gatewayv1alpha2.GRPCRoute, modified *gatewayv1alpha2.GRPCRoute) (ctrl.Result, error) {
 	if err := r.updateStatus(ctx, original, modified); err != nil {
-		return controllerruntime.Fail(fmt.Errorf("failed to update GRPCRoute status while handling the reconcile error %w: %w", reconcileErr, err))
+		return controllerruntime.Fail(fmt.Errorf("failed to update GRPCRoute status while handling the reconcile error: %w: %w", reconcileErr, err))
```

Comment:
nice, I didn't know that we can wrap multiple errors at all.

Reply:
Works since Go 1.20 :-)